tests/Unit/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Test_DampedHarmonicGaugeQuantities.cpp

```diff
@@ -148,10 +148,34 @@ tnsr::a<DataVector, SpatialDim, Frame> wrap_DampedHarmonicHCompute(
     const double t, const double t_start, const double sigma_t,
     const tnsr::I<DataVector, SpatialDim, Frame>& coords,
     const double sigma_r) noexcept {
-  return GeneralizedHarmonic::gauges::DampedHarmonicHCompute<
-      SpatialDim, Frame>::function(gauge_h_init, lapse, shift,
-                                   sqrt_det_spatial_metric, spacetime_metric, t,
-                                   t_start, sigma_t, coords, sigma_r);
+  tnsr::a<DataVector, SpatialDim, Frame> gauge_h{};
+  tnsr::ab<DataVector, SpatialDim, Frame> d4_gauge_h{};
+  const auto dgauge_h_init =
+      make_with_value<tnsr::ab<DataVector, SpatialDim, Frame>>(lapse, 0.);
```

Comment:
Why not initialize with signaling NaNs here?

Reply:
This one I could do, though it'll be rewritten in a followup PR. The reason the ones below can't be `NaN` is because they are used inside the `damped_harmonic` function to compute the spacetime derivative. If you want I'm happy to make `dgauge_h_init` signaling NaN.

Reply:
Up to you